controllers/admin/AdminSearchConfController.php

```diff
@@ -106,11 +115,18 @@ public function __construct()
 					</a><br /><br />
 					<p>
 						' . $this->trans('You can set a cron job that will rebuild your index using the following URL:', [], 'Admin.Shopparameters.Feature') . '<br />
-						<a href="' . Tools::safeOutput($cron_url) . '">
+						<a href="' . Tools::safeOutput($cronUrlFullIndex) . '">
 							<i class="icon-external-link-sign"></i>
-							' . Tools::safeOutput($cron_url) . '
+							' . Tools::safeOutput($cronUrlFullIndex) . '
 						</a>
-					</p><br />',
+					</p>
+                    <p>
+                        ' . $this->trans('You can set a cron job that will index your missing products using the following URL:', [], 'Admin.Shopparameters.Feature') . '<br />
```

Comment:
Could you check the wording @l-delin ? It is a simple differentiation between a full indexation and indexing only the missing products :)

Reply:
![image](https://user-images.githubusercontent.com/70583503/190670681-fec61619-f891-422c-a2e0-b4b73049f341.png)
Hi @kpodemski. There's something I don't understand: doesn't this option already exist?

Reply:
The one that exists is about reindexing everything, and this PR is about adding the second option to index what's missing only